server/stressTest/stressTestHelper.js

```diff
@@ -0,0 +1,26 @@
+function getRandom(userContext, events, done) {
+  const itemId = Math.floor(Math.random() * 10000000 + 1);
+
+  userContext.vars.itemId = itemId;
```

Comment:
This looks interesting. Could you tell me a bit more about this line and the .vars?

Reply:
This is an Artillery-specific thing, as far as I'm aware. You can call functions within the "scenario" part of your Artillery test, and that function is given 3 arguments: the virtual user’s context, an EventEmitte, and a callback.